src/project_types.rs

```diff
@@ -1,17 +1,38 @@
 /// Structure to represent a Proton Project.
 /// This is what will be written to a Protonfile at the project root.
```

Comment:
This should be right next to Project. Move User down below `pub struct Project` and above `impl Project`.